code/src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+import { list } from 'reducers/todolist'
+import { headerStyle } from 'components/headerStyle.css'
+import { ListSummary } from './ListSummary'
+
+export const Header = () => {
+  
+return (
+    <section className='header'>
+        <h1>To-do-list</h1>
+    </section>  
+)
```

Comment:
Here you can use an implicit return like this:
```
export const Header = () => (
    <section className='header'>
        <h1>To-do-list</h1>
    </section>  
)
```

Read more about[ explicit vs implicit returns here](https://dev.to/samanthaming/es6-arrow-functions-cheatsheet-1cn)